backup/queries_functions.go

```diff
@@ -831,13 +831,37 @@ func (e Extension) FQN() string {
 func GetExtensions(connectionPool *dbconn.DBConn) []Extension {
 	results := make([]Extension, 0)
 
-	query := fmt.Sprintf(`
-	SELECT e.oid,
-		quote_ident(extname) AS name,
-		quote_ident(n.nspname) AS schema
-	FROM pg_extension e
-		JOIN pg_namespace n ON e.extnamespace = n.oid
-	WHERE e.oid >= %d`, FIRST_NORMAL_OBJECT_ID)
+	query := ""
+	if connectionPool.Version.Before("6") {
+		query = fmt.Sprintf(`
```

Comment:
You do fmt.Sprintf in both branches, so it can be optimized and moved out of if statement.